src/apps/argo-finance/cronos/argo-finance.x-argo.token-fetcher.ts

```diff
@@ -28,7 +28,10 @@ export class CronosArgoFinancePledgingTokenFetcher implements PositionFetcher<Ap
   async getVePosition(address: string, baseAddress: string) {
     const multicall = this.appToolkit.getMulticall(network);
     const baseTokens = await this.appToolkit.getBaseTokenPrices(network);
-    const baseToken = baseTokens.find(t => t.address === baseAddress)!;
+    let baseToken = baseTokens.find(t => t.address === baseAddress)!;
+    baseToken.address = address
+    baseToken.symbol = 'xARGO'
```

Comment:
Not necessary, the underlying token _is_ `ARGO`

Reply:
(the `tokens` array is intended to represent the underlying token, and here, you'd want the ARGO underlying in xARGO)